docs/learn/learn-xcm.md

```diff
@@ -7,28 +7,60 @@ keywords: [cross-consensus, XCM, XCMP, interoperability, communication]
 slug: ../learn-xcm
 ---
 
-Cross-Consensus Message Format (XCM) aims to be a language to communicate ideas between consensus
-systems. One of {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}'s main initiatives is
-that of interoperability, and XCM is the vehicle through which it will deliver this promise. Simply,
-it is a standard that allows protocol developers to define the data and origins which their chains
-can send and receive from. Out of the box, it comes with a VM that allows for customization of
-execution as well as the following properties:
+The Cross-Consensus Message Format, or **XCM**, is a **messaging format** and language used to
+define messages between consensus systems.
```

Comment:
Does XCM define the messages or does it provide guidelines on how to construct them such that the communication can happen between different consensus systems? Just added an edit that made sense to me. Feel free to rephrase or improve on introducing it. I think the previous author took that sentence from here https://polkadot.network/blog/xcm-the-cross-consensus-message-format#-goals

```suggestion
The Cross-Consensus Message Format, or **XCM**, is a **messaging format** and language used to
communicate between consensus systems.
```

Reply:
It technically does both, as it *defines* the format in which messages are constructed and executed. However in this context,  `communicate` works better 👍